Documentation/ceph-configuration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuration
+weight: 3700
+indent: true
+---
+
+# Configuration
+
+For most any Ceph cluster, the user will want to -- and may need to -- change some Ceph
+configurations. These changes often may be warranted in order to alter performance to meet SLAs or
+to update default data resiliency settings.
+
+**WARNING:** Modify Ceph settings carefully, and review the
+[Ceph configuration documentation](https://docs.ceph.com/docs/master/rados/configuration/) before
+making any changes. Changing the settings could result in unhealthy daemons or even data loss if
+used incorrectly.
+
+
+## Required configurations
+Rook and Ceph both strive to make configuration as easy as possible, but there are some
+configuration options which users are well advised to consider for any production cluster.
+
+### Default PG and PGP counts
```

Comment:
With #3769 we could add the recommendation here to enable the autoscale module. #3769 also makes it the default, assuming we go with that. But if they don't enable it, this section is still very helpful to configure for PGs.

Reply:
Is #3769 feature creeping our release?

Reply:
How about we get this PR in first and discuss #3769 separately.